Doc/library/pdb.rst

```diff
@@ -211,6 +211,10 @@ access further features, you have to do this yourself:
    The *readrc* argument defaults to true and controls whether Pdb will load
    .pdbrc files from the filesystem.
 
+   The *invoke_origin* argument is used to determine the origin of the debugger.
```

Comment:
I think it should be `invocation_origin` rather than `invoke_origin`. There are probably rules about this somewhere in the docs.